sphinxcontrib/websupport/builder.py

```diff
@@ -178,6 +178,25 @@ def handle_finish(self):
                     shutil.rmtree(dst)
                 shutil.move(src, dst)
 
+        # copy resource files to static dir
```

Comment:
Could you move this block to a new method please?

Reply:
I fixed it.